app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -46,8 +46,8 @@ enum class NavigationItem(
     internal val primaryRoute: String,
     private val canonicalRoute: String,
     val arguments: List<NamedNavArgument> = emptyList(),
-    open val content: @Composable (ContentParams) -> Unit
-    // TODO add animations here
+    open val content: @Composable (ContentParams) -> Unit,
+    open val animationConfig: NavigationAnimationConfig = NavigationAnimationConfig.NoAnimationConfig
```

Comment:
why is this open ?

Reply:
Good point, it was a left over from another approach I was taking before going with this, removed 👍